cdn_lambda/functions/map_to_s3/map_to_s3.py

```diff
@@ -63,6 +63,11 @@ def handler(self, event, context):
             LOG.info("Item found for '%s'", request["uri"])
 
             try:
+                # Add custom header containing the original request uri
+                request["headers"]["original-uri"] = [
```

Comment:
Just confirming, have you already deployed this in some environment and seen it working?

Also, could we please use something which stands out more for our own custom headers; how about a standard prefix, like "exodus-" ?

At a glance, if someone is looking at code referencing an "original-uri" header, it won't be obvious that this is something we invented, while "exodus-original-uri" makes it clear.  Also it can't be ruled out that a future version of the CloudFront platform would introduce its own original-uri header, as the name is fairly generic.

Reply:
> Just confirming, have you already deployed this in some environment and seen it working?

Yes. I deployed this in my 'caleigh' cloudfront distribution with the 'origin_reponse' lambda function.